### ISSUE TITLE
ParseSwift.podspec s.version 1.0.0

### ISSUE THREAD

**Issue by softcleandev** [netreconlab/Parse-Swift#81]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/netreconlab/Parse-Swift/security/policy).
- [x] I have searched through [existing issues](https://github.com/netreconlab/Parse-Swift/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest versions of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Swift SDK](https://github.com/netreconlab/Parse-Swift/releases). 

### Issue Description

Pod version of Parse-Swift (OG) is always 1.0.0 and it's a bit irritating.

### Steps to reproduce

- Update Parse-Swift via podfile 
Here the Podfile entry
`  pod 'ParseSwift', :git => 'https://github.com/netreconlab/Parse-Swift.git', :branch => 'main'` 
[Installation via Cocoapods](https://github.com/netreconlab/Parse-Swift#cocoapods)
- Execute `pod update` in the shell (zsh, bash etc.)

### Actual Outcome

```
dev % pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
Pre-downloading: `ParseSwift` from `https://github.com/netreconlab/Parse-Swift.git`, branch `main`
Downloading dependencies
Installing ParseSwift 1.0.0
```

### Expected Outcome

I expect to see the correct version in the output like:

```
dev % pod update
Update all pods
Updating local specs repositories
Analyzing dependencies
Pre-downloading: `ParseSwift` from `https://github.com/netreconlab/Parse-Swift.git`, branch `main`
Downloading dependencies
Installing ParseSwift 5.3.0
```

### Environment

Client
- Parse Swift SDK version: `5.3.0`
- Xcode version: `14.2`
- Operating system (iOS, macOS, watchOS, etc.): `macOS`
- Operating system version: `Ventura 13.2.1`

---

I'm not sure, but may be the s.version value in ParseSwift.podspec should be changed with every release etc.

**Comment by cbaker6** [comment]
Even though it says 1.0.0, does your code update to the latest? What cocoa pods says isn’t as important as what [ParseVersion.current()](https://swiftpackageindex.com/netreconlab/parse-swift/5.3.0/documentation/parseswift/parseversion) says in your code

**Comment by softcleandev** [comment]
Source code is 5.3.0 and ParseVersion.current() returns 5.3.0.

It's only the `pod update/install` command, that shows the 1.0.0 version.

**Comment by cbaker6** [comment]
This probably won't be fixed, you can see my comment here: https://github.com/netreconlab/Parse-Swift/pull/19#issuecomment-1344620039. There is a naming clash when attempting to push this to cocoapods. Time wise, I plan to support SPM directly as that is a lot simpler and faster to manage than cocoapods, since the pod file currently works, I don't believe the name clash is worth the time as most people will use SPM.

**Comment by softcleandev** [comment]
ok. time to switch to SPM 👍

**Comment by cbaker6** [comment]
ParseSwiftOG has been released in `5.3.1`, though I haven't tried it myself as I haven't used coocoapods in years. You can follow the directions here to add to your project: https://github.com/netreconlab/Parse-Swift#cocoapods. The new podspec shows on cocoapods here: https://github.com/CocoaPods/Specs/commit/b37513b6dccde2ac2fe62dafbfb61c7330837b4c

Keep in mind, you might have to change the project to use `import ParseSwiftOG` instead of `import ParseSwift` when using Cocoapods.

**Comment by softcleandev** [comment]
> 

already changed it to ParseSwiftOG and successfully tested `pod update`.
well done!
